### PR TITLE
Integrated KumuluzEE v3.2

### PR DESCRIPTION
**Pull request by zvonegit**
Integrated KumuluzEE v3.2 with MP 1.2, 1.3, 1.4, 2.0 and 2.1.

### CODE REVIEW

File: src/main/resources/pom-servers.xml [hunk 1]
```diff
@@ -623,6 +623,198 @@
 
         </profile>
 
+        <profile>
+            <id>kumuluzEE-1.3</id>
+
+            <properties>
+                <kumuluz.version>3.2.0</kumuluz.version>
```
Comment (rdebusscher) [inline src/main/resources/pom-servers.xml]: Should that not be version KumuluzEE version 2.6.0 which is MP 1.3 compliant.

File: src/main/resources/pom-servers.xml [hunk 2]
```diff
@@ -579,8 +579,8 @@
             <id>kumuluzEE-1.2</id>
 
             <properties>
-                <kumuluz.version>2.5.2</kumuluz.version>
-                <kumuluzee-config-mp.version>1.1.1</kumuluzee-config-mp.version>
+                <kumuluz.version>3.2.0</kumuluz.version>
```
Comment (rdebusscher) [inline src/main/resources/pom-servers.xml]: We do not have the habit of updating the versions for older MicroProfile versions.

File: src/main/resources/pom-servers.xml [hunk 3]
```diff
@@ -623,6 +623,198 @@
...
+                </plugins>
+            </build>
+
+        </profile>
+
+        <profile>
+            <id>kumuluzEE-1.4</id>
+
+            <properties>
+                <kumuluz.version>3.2.0</kumuluz.version>
```
Comment (rdebusscher) [inline src/main/resources/pom-servers.xml]: Should that not be version KumuluzEE version 2.6.0 which is MP 1.4 compliant.

File: src/main/resources/pom-servers.xml (hunk 1 above)
Comment (zvonegit) [reply src/main/resources/pom-servers.xml]: Updated

File: src/main/resources/pom-servers.xml (hunk 2 above)
Comment (zvonegit) [reply src/main/resources/pom-servers.xml]: Reverted to previous

File: src/main/resources/pom-servers.xml (hunk 3 above)
Comment (zvonegit) [reply src/main/resources/pom-servers.xml]: Updated

**Pull request merged**